service/species-service.js

```diff
@@ -0,0 +1,28 @@
+
+const axios = require('axios')
+const log = global.log
+const ADMIN_BASE_API_URL = `${process.env.ADMIN_BASE_API_URL}`
+const SERVICE_NAME = 'species-service'
+
+async function createSpecies (payload) {
+  try {
+    let url = ADMIN_BASE_API_URL + '/species/product'
+    log.info(`${SERVICE_NAME}::createSpecies:url:${url}`)
+    return axios.post(url, payload)
+  } catch (error) {
+    log.error(error)
+  }
+}
+
+async function deleteSpecies (productNo) {
+  try {
+    let url = ADMIN_BASE_API_URL + `/species/productNo/${productNo}`
```

Comment:
productNo?

Reply:
The species is associated with a product. We are going to allow a user to register a new product and at the same time a species. There are no checks to on the species name apparently. Could be anything!

Hence, we can only get to the species using the referenced product number. Using the name wouldn't be unique.